0x48piraj/app.py

```diff
@@ -0,0 +1,50 @@
+from flask import Flask, render_template, flash, request
+from wtforms import Form, TextField, TextAreaField, validators, StringField, SubmitField
+import sys
+import json
+import os
+import shutil
+from os import path
+import urllib.request
+from bs4 import BeautifulSoup
+from scraper import AdvancedSearchScraper
+
+path = path.dirname(path.realpath(__file__))
+app = Flask(__name__,template_folder=path+'/')
+app.config['SECRET_KEY'] = '3d441f27u331c27333d331k2f3333a'
```

Comment:
@0x48piraj  Kindly don't commit your secret key. You can store and use it through environment variable.
This might help you though https://blog.fossasia.org/setting-environment-variables-up-in-travis-and-heroku-for-meilix-and-meilix-generator/

Reply:
Wow! Thanks.
Will do soon.